stdlib/2.7/__builtin__.pyi

```diff
@@ -879,3 +880,15 @@ class file(BinaryIO):
     def write(self, data: str) -> None: ...
     def writelines(self, data: Iterable[str]) -> None: ...
     def truncate(self, pos: int = ...) -> int: ...
+
+def apply(object: object, *args: Iterable, **kwargs: Dict[str, Any]) -> Any: ...
+
+def coerce(x: _Number, y: _Number) -> Tuple[_Number,_Number]: ...
+
+def help(str) -> None: ...
```

Comment:
Can you give the argument a type (should be Any or object) and a different name than `str`?